graylog2-web-interface/src/components/event-definitions/event-definitions/EventDefinitionEntry.tsx

```diff
@@ -76,6 +79,11 @@ const EventDefinitionEntry = ({
 }: Props) => {
   const [showEntityShareModal, setShowEntityShareModal] = useState(false);
   const isScheduled = lodash.get(context, `scheduler.${eventDefinition.id}.is_scheduled`, true);
+  const { isLoading, data } = useGetPermissionsByScope(eventDefinition?._scope || 'DEFAULT');
```

Comment:
The hook should be taking an entity, so the logic to find where the scope gets encapsulated in the hook.